src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/NativeToManagedStubCodeContext.cs

```diff
@@ -61,7 +61,6 @@ public override (string managed, string native) GetIdentifiers(TypePositionInfo
             // As a result, we generate another name for the native return value.
             if (info.IsManagedReturnPosition)
             {
-                Debug.Assert(info.NativeIndex == TypePositionInfo.UnsetIndex);
```

Comment:
The comment above this line makes this assert seem correct. Since that is the case, I would expect removing the assert warrants updating the comment.